cpp/collections_module/collections.cpp

```diff
@@ -1,9 +1,22 @@
 #include <mgp.hpp>
 
+#include "algorithm/algorithm.hpp"
+
+constexpr const char *kProcedureAvg = "avg";
+
+constexpr const char *kReturnAvg = "average";
+
+constexpr const char *kNumbersList = "list_of_numbers";
```

Comment:
use on these `constexpr std::string_view` as` const char * `is a global that may change.

Reply:
I would move these globals to hpp if possible, otherwise keep all either in cpp or hpp.